SPECS/kernel/kernel.spec

```diff
@@ -413,6 +413,10 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Fri Dec 01 2023 Cameron Baird <cameronbaird@microsoft.com> - 6.1.58.1-2
+- Remove loglevel=3, causing kernel to boot with the config-defined value,
+    https://cateee.net/lkddb/web-lkddb/CONSOLE_LOGLEVEL_DEFAULT.html
```

Comment:
Remove this link out. No guarantee link will continue to exist in the future, and cateee.net, while a helpful resource, is not the authoritative source on kernel configs (which would be the linux kernel documentation)